src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -228,8 +232,21 @@ public void setIgnoreCommitList(List<CommitHash> ignoreCommitList) {
         return authorList;
     }
 
+    public void setAuthorDetails(Author author) {
```

Comment:
Header comments for this method

Reply:
Can I know what the header comment means? Do you mean javadoc for this method? Or a simple line of word explaining what this method do?

Reply:
Yes javadocs